src/dotnet/commands/dotnet-compile-native/IntermediateCompilation/Linux/LinuxCppCompileStep.cs

```diff
@@ -25,7 +25,7 @@ public class LinuxCppCompileStep : IPlatformNativeStep
 
         private readonly string[] _appdeplibs = 
             {
-                "libSystem.Native.a"
+                "System.Native.a"
```

Comment:
Yes. It has changed to System.Native.a. We will soon consume that as well from the dependencies (through NETStandard.Library) I don't think it is wired up that way right now.

Cc @tarekgh